### DISCUSSION TITLE
about kubeedge  cloudcore HA

### DISCUSSION THREAD

**Discussion by davidleecn** [kubeedge/kubeedge#5293] [Q&A]
hi，

I have some confusion about about the coudcore HA，I saw the official docs about "Deploying HA CloudCore" ,as it mentions ,the cloudcore HA will be supported by ingress later. becuz I deploy cloudcore through Helm on my k8s cluster，so I wanna know if i can deploy multiple pods to different nodes ,and i enable the Nodeport service type of cloudcore service ,then I use one loadbalancer to proxy the k8s node,the ip of the LB will be used as the  "advertiseAddress" of the cloudcore for edge node connection。

in summary,the flow will be:  edge node -> LB ip(**advertiseAddress**) -> k8s node(nodePort service) ->cloudcore pod (**multiple pods on different k8s nodes**).

is the design above resonable ? mutileple cloudcore pods are permitted?

tks a lot !
